### DISCUSSION TITLE
Homeschool

### DISCUSSION THREAD

**Discussion by ZiroEGay** [community/community#45362] [GitHub Education]
Hi, I am homeschooled and from Europe.
I'm trying to sign up for the github student package, but it wants documents that I don't have...
I don't go to school, I am homeschooled, so I don't have any school ids/documents, what should I put here?

![image](https://user-images.githubusercontent.com/88873885/214638679-036b34bb-84aa-499d-8a5e-484d44a17d03.png)

*Btw I am 15 years old and the only document I have is my ID.*

**Comment by VladCuciureanu** [comment]
Hey! Sadly one of the requirements for Github Student Pack is the following:
- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution.

From my own experience, GitHub is pretty serious about this and having the documents up to date so most likely you won't be able to apply but, as a last resort, you could message support via email and try your luck.
